src/Tasks/Microsoft.Build.Tasks.csproj

```diff
@@ -970,6 +970,9 @@
     <PackageReference Include="System.Resources.Writer" Version="4.0.0" />
     <PackageReference Include="System.Threading.Tasks.Dataflow" Version="4.6.0" />
 
+    <!-- Need Win32 API on .NET Core to ping registry to determine long path support -->
+    <PackageReference Include="Microsoft.Win32.Registry" Version="$(MicrosoftWin32RegistryVersion)" />
```

Comment:
Can these in-every-assembly requirements be moved to https://github.com/Microsoft/msbuild/blob/master/src/Directory.Build.props, or is that too generic? It seems a chatty to mention them in every project.